easy_study/lib/view/SubjectProgressBar.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:easy_study/model/Subject.dart';
+import 'package:easy_study/presenter/TimeTracking.dart';
+import 'package:easy_study/store/AppState.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:redux/redux.dart';
+
+class SubjectProgressBar extends StatefulWidget {
```

Comment:
Is a StatefulWidget really needed, isn't StatelessWidget enough?

Reply:
you cant make it stateless because you would need to implement the build method directly.
But having the build method in a private subclass is the given architecture which i copied from the subject card list.

To answer the question: no it is not enough ;=)